.github/workflows/python-package-ci.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python Package CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
```suggestion
  push:
  pull_request:
```

I recommend removing the branch restriction.

That way contributors can create a feature branch and test their code on CI _before_ creating a PR.

Then they can be confident their changes pass, and don't need to create a PR to test on the CI, and go through more steps to get things passing.